Dockerfile

```diff
@@ -1,6 +1,16 @@
 FROM olantwin/ship-base:20180627-online
 
-ADD . /FairShip
+# Copy FairShip scripts
+COPY . /FairShip
 
-RUN aliBuild -c shipdist/ --defaults fairship build FairShip --no-local ROOT \
-	&& aliBuild clean --aggressive-cleanup
+# Build FairShip
+RUN aliBuild -c shipdist/ --defaults fairship build FairShip --no-local ROOT
+
+# Additional library for OpenGL
+RUN yum install -y mesa-dri-drivers
+
+# Fix problems with fonts
+RUN yum install -y dejavu-lgc-sans-fonts
```

Comment:
I think adding the packages should happen in the base image to minimise build times: <https://github.com/olantwin/ship-base>

Reply:
Agree. 

Ship-base Dockerfile is great. Can you add this link into the FairShip Dockerfile or ship-base on dockerhub? It is very useful.

Reply:
I hope we can also move it to ShipSoft eventually (@FonsRademakers ?)

I'll create a pull-request to your repo adding the info. (and should probably add some more info on Dockerhub as well)

Reply:
Also a good time to maybe rebuild ship-base...